File: bronze/10813/python/main.py

if (__name__ == "__main__"):
    n, m = map(int, input().split())
    array = [_ for _ in range(1, n +1)]

    for _ in range(m):
        i, j = map(int, input().split())
        temp = array[i -1]
        array[i -1] = array[j -1]
        array[j -1] = temp
    for data in array:
        print(data, end=' ')

File: bronze/10813/c++/main.cpp
#include <iostream>

int main()
{
    int n, m, i, j, temp;
    std::cin >> n >> m;

    int array[n];

    for (int i = 0; i < n; i++) {
        array[i] = i +1;
    }

    for (int count = 0; count < m; count++) {
        std::cin >> i >> j;
        temp = array[i - 1];
        array[i - 1] = array[j - 1];
        array[j - 1] = temp;
    }

    for (int data : array) {
        std::cout << data << ' ';
    }

    return 0;
}